### PR TITLE
Fix security setting idempotentance

### PR DESCRIPTION
**Pull request by elconas**
fixes https://github.com/jenkinsci/puppet-jenkins/issues/673

### CODE REVIEW

**Comment by roidelapluie** [comment]
:-1: This is not working; still reload is triggirid

File: Gemfile [hunk 1]
```diff
@@ -16,6 +16,7 @@ gem 'rubocop', '< 0.42.0'
 
 gem 'json_pure', '< 2.0.2'
 gem 'net-http-persistent', '<= 2.9.4', :require => false if RUBY_VERSION < '2.0.0'
+gem 'addressable', '<= 2.4.0', :require => false if RUBY_VERSION < '2.0.0'
```
Comment (elconas) [inline Gemfile]: To fix spec errors as the latest adressable (2.5.0) needs ruby 2.0 and then the 1.9 integration tests on Travis failes.

**Comment by rtyler** [comment]
@jhoblitt this looks reasonable to me, any downsides to merging?

File: Gemfile (hunk 1 above)
Comment (jhoblitt) [inline Gemfile]: Please remove this.  Ruby < 2 has been dropped from the travis matrix.

**Comment by jhoblitt** [comment]
@rtyler The Gemfile change should be dropped.  Otherwise, it looks good to me.

**Pull request closed without merging**